datumaro/util/annotation_util.py

```diff
@@ -266,10 +266,12 @@ def make_label_id_mapping(
     labels.
 
     Returns:
-      map_id (callable): src id -> dst id
-      id_mapping (dict): src id -> dst id
-      src_labels (dict): src id -> src label
-      dst_labels (dict): dst id -> dst label
+        .. code-block::
+
+        map_id (callable): src id -> dst id
+        id_mapping (dict): src id -> dst id
+        src_labels (dict): src id -> src label
+        dst_labels (dict): dst id -> dst label
```

Comment:
I'm not sure it is a good change. Can we just have a few lines without a code block?

Reply:
Places where there are multiple return types are fixed by adding `Returns:` for each

Reply:
I can't say it's really clear now:

![image](https://user-images.githubusercontent.com/13832349/156629583-47130bba-00c2-45f2-8d5f-4ebd27545359.png)

Probably, it will be better to just have a code block then.

Reply:
![image](https://user-images.githubusercontent.com/54434686/156777271-783cdf85-982f-479d-8640-825d3d2b6450.png)
![image](https://user-images.githubusercontent.com/54434686/156777303-109293f3-eff1-4cbc-a749-4f12a7c9b557.png)
Maybe do it this way?

Reply:
Probably, I'm ok with this variant.